src/Altinn.Broker.API/Models/FileTransferUploadResponseExt.cs

```diff
@@ -0,0 +1,6 @@
+namespace Altinn.Broker.API.Models;
+
+public class FileTransferUploadResponseExt
+{
+    public Guid FileId { get; set; }
```

Comment:
Consider correcting this naming.
"fileId" is an outdated term, the correct name is "fileTransferId"
(In this case the property name was not previousy exposed in the API, but with this change it becomes; so therefore naming becomes importan).